hybris/glesv2/glesv2.c

```diff
@@ -48,6 +48,133 @@ static void         (*_glVertexAttrib3f)(GLuint indx, GLfloat x, GLfloat y, GLfl
 static void         (*_glVertexAttrib4f)(GLuint indx, GLfloat x, GLfloat y, GLfloat z, GLfloat w) FP_ATTRIB = NULL;
 static void         (*_glEGLImageTargetTexture2DOES)(GLenum target, GLeglImageOES image) = NULL;
 
+static void         (*_glActiveTexture)(GLenum texture) = NULL;
+static void         (*_glAttachShader)(GLuint program, GLuint shader) = NULL;
+static void         (*_glBindAttribLocation)(GLuint program, GLuint index, const GLchar* name) = NULL;
```

Comment:
It seems original sym list is already sorted by name? So probably these proposed new symbols should be merged into the list instead.